pubspec.yaml

```diff
@@ -1,98 +1,47 @@
-name: fluttercon_2024
+name: fluttercon2024
 description: "A new Flutter project."
-# The following line prevents the package from being accidentally published to
-# pub.dev using `flutter pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
-
-# The following defines the version and build number for your application.
-# A version number is three numbers separated by dots, like 1.2.43
-# followed by an optional build number separated by a +.
-# Both the version and the builder number may be overridden in flutter
-# build by specifying --build-name and --build-number, respectively.
-# In Android, build-name is used as versionName while build-number used as versionCode.
-# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
-# In iOS, build-name is used as CFBundleShortVersionString while build-number is used as CFBundleVersion.
-# Read more about iOS versioning at
-# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-# In Windows, build-name is used as the major, minor, and patch parts
-# of the product and file versions while build-number is used as the build suffix.
+publish_to: 'none'
 version: 1.0.0+1
 
 environment:
   sdk: '>=3.3.1 <4.0.0'
 
-# Dependencies specify other packages that your package needs in order to work.
-# To automatically upgrade your package dependencies to the latest versions
-# consider running `flutter pub upgrade --major-versions`. Alternatively,
-# dependencies can be manually updated by changing the version numbers below to
-# the latest version available on pub.dev. To see which dependencies have newer
-# versions available, run `flutter pub outdated`.
 dependencies:
+  cupertino_icons: ^1.0.6
   flutter:
     sdk: flutter
-
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^1.0.6
-  dio: ^5.4.3+1
+  flutter_bloc: ^8.1.5
+  flutter_localizations:
+    sdk: flutter
+  http: ^1.2.1
+  get_it: ^7.7.0
+  injectable: ^2.4.2
+  intl: ^0.18.0
   freezed_annotation: ^2.4.1
   json_annotation: ^4.9.0
-  mocktail: ^1.0.3
+  rxdart: ^0.27.7
+  shared_preferences: ^2.2.3
+  styled_widget: ^0.4.1
+  textstyle_extensions: ^2.0.0-nullsafety
+  equatable: ^2.0.5
+  stream_transform: ^2.1.0
+  bloc_concurrency: ^0.2.5
+  flutter_svg: ^2.0.10+1
 
 dev_dependencies:
+  build_runner: ^2.4.6
+  flutter_launcher_icons: ^0.13.1
+  flutter_lints: ^3.0.0
   flutter_test:
     sdk: flutter
+  freezed: ^2.3.5
+  injectable_generator: ^2.1.6
+  json_serializable: ^6.7.0 
+  mockito: ^5.0.15
```

Comment:
Any reason for preferring mockito over mocktail? The current version uses mocktail which doesn't rely on code generation.

We aren't married to the approach, just curious on why you'd replace mocktail with mockito

Reply:
I just added what I work with daily but am open to what you guys have set in place